api/src/test/java/org/pmiops/workbench/api/AuthDomainControllerTest.java

```diff
@@ -51,15 +52,17 @@
   private static final String CURRENT_POSITION = "Tester";
   private static final String RESEARCH_PURPOSE = "To test things";
 
+  @Autowired private UserDao userDao;
+
+  @Mock private AccessTierService accessTierService;
```

Comment:
These seemingly-unrelated tests need to include an `AccessTierService` because they use `UserService` which now requires it.